code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import "./moviedetails.css"
+
+// In useState initial state is set to 2 object rendered on first mount while page is loading
+export const MovieDetails = () => {
+  const api_key = "bf1c38c5c54137f389c5c7971de1fe96"
```

Comment:
Usually you want to store sensitive information like this outside of the github project.  This prevents other people seeing your code and using your api_key. 

One way to do this is storing it in a file which is not committed to github.  Another way is to use `process.env.API_KEY` and pass the value at runtime -- or store it in a `.env`  file.

You (hopefully) won't see many api_keys in production code